android/java/org/chromium/chrome/browser/BraveRewardsPublisher.java

```diff
@@ -13,10 +13,9 @@ public class BraveRewardsPublisher {
     //PublisherStatus @
     //vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom
     @Retention(RetentionPolicy.SOURCE)
-    @IntDef({NOT_VERIFIED, CONNECTED, UPHOLD_VERIFIED, BITFLYER_VERIFIED, GEMINI_VERIFIED})
+    @IntDef({NOT_VERIFIED, UPHOLD_VERIFIED, BITFLYER_VERIFIED, GEMINI_VERIFIED})
```

Comment:
we should remove this IntDef altogether. instead we should use mojom struct for the same.